include/scan_class/scan_part_dyn.klc

```diff
@@ -152,18 +153,20 @@ ROUTINE log_pose
     origin = (ZEROPOS(1))
 
     --get position
-    p = `MOTION_DATA_FILE`__get_cart(rbt_pos)
+    -- p = `MOTION_DATA_FILE`__get_cart(rbt_pos)
+    $UFRAME = $MNUFRAME[CD_PAIR,USER_FRAME_NO]
+    $UTOOL = $MNUTOOL[CD_PAIR,TOOL_FRAME_NO]
+    p = CURPOS(0, 0, RBT_GRP)
```

Comment:
This is fine, but I would prefer to use the motion interface most of the time as not to run into issues when porting to different robot configuration (i.e. groups).  
This should do the same thing as above:
```
`MOTION_DATA_FILE`__set_lpos
`MOTION_DATA_FILE`__get_cart(rbt_pos)
```

the other benefit is

Reply:
Thank you for the suggestion.